### ISSUE TITLE
Add a doc for setting up AWS SSO

### ISSUE THREAD

**Issue by praneetloke** [pulumi/pulumi-hugo#1071]
...it would also be nice to have a doc that explains the various attributes that users are asked to configure.

**Comment by interurban** [comment]
hiya; We've created specific docs guides now for https://www.pulumi.com/docs/pulumi-cloud/oidc/ providers, and a specific guide for AWS https://www.pulumi.com/docs/pulumi-cloud/oidc/aws/ and example code to make this easy to configure : https://github.com/pulumi/examples/tree/master/aws-py-oidc-provider-pulumi-cloud . thank you!

**Comment by ericrudder** [comment]
@interurban ... I don't think this is at all what @praneetloke was asking for ... I think this request is for SAML settings

**Comment by praneetloke** [comment]
That's right. My [comment](https://github.com/pulumi/pulumi-hugo/issues/1071#issuecomment-1049153782) above clarifies this a bit. Admittedly, the issue title wasn't clear that this was for SAML. I've updated it now.

Glad that a doc exists for configuring AWS for OIDC though. That's very helpful!
